offchain/authority-claimer/src/lib.rs

```diff
@@ -8,64 +8,69 @@ pub mod listener;
 pub mod metrics;
 pub mod sender;
 
+#[cfg(test)]
+mod mock;
+
 use config::Config;
 use rollups_events::DAppMetadata;
 use snafu::Error;
 use tracing::trace;
 
 use crate::{
     checker::DefaultDuplicateChecker,
-    claimer::{AuthorityClaimer, DefaultAuthorityClaimer},
-    listener::DefaultBrokerListener,
+    claimer::AbstractClaimer,
```

Comment:
If we move the main loop back to the claimer, we will probably have to revert back all these modifications.

Reply:
Done at https://github.com/cartesi/rollups-node/commit/226c386d96f9c8445dea95a31c03455991bc9114